README.rst

```diff
@@ -24,7 +24,7 @@ See also:
 Installation
 ============
 
-To install the latest release on `PyPi <https://pypi.python.org/pypi/toml/0.9.2>`_,
+To install the latest release on `PyPi <https://pypi.org/project/toml/0.9.4/>`_,
```

Comment:
Hm. Is there a reason this should really be linking to a specific version at all?